### DISCUSSION TITLE
Receiving exception during vsix installation (wasn't showing up in Extension manager)

### DISCUSSION THREAD

**Discussion by ColinStroud** [community/community#15164] [Copilot Conversations]
Edit: If there is a better place for this to live, please let me know.

VSIX Version: 1.12.0.0

![VSIX Error](https://user-images.githubusercontent.com/41063766/163912129-523ffee4-c08b-442a-998b-eb4a1ef288d8.png)

Error Log / Stack Trace:

4/18/2022 10:02:21 PM - Microsoft VSIX Installer
4/18/2022 10:02:21 PM - -------------------------------------------
4/18/2022 10:02:21 PM - vsixinstaller.exe version:
4/18/2022 10:02:21 PM - 17.0.5226-preview5
4/18/2022 10:02:21 PM - -------------------------------------------
4/18/2022 10:02:21 PM - Command line parameters:
4/18/2022 10:02:21 PM - C:\Program Files (x86)\Microsoft Visual Studio\Installer\resources\app\ServiceHub\Services\Microsoft.VisualStudio.Setup.Service\VSIXInstaller.exe,C:\Users\Colin\Downloads\GitHub.Copilot.Vsix.1.12.0.0.vsix
4/18/2022 10:02:21 PM - -------------------------------------------
4/18/2022 10:02:21 PM - Microsoft VSIX Installer
4/18/2022 10:02:21 PM - -------------------------------------------
4/18/2022 10:02:22 PM - Initializing Install...
4/18/2022 10:02:22 PM - Extension Details...
4/18/2022 10:02:22 PM - 	Identifier         : GitHub.Copilot
4/18/2022 10:02:22 PM - 	Name               : GitHub Copilot
4/18/2022 10:02:22 PM - 	Author             : GitHub
4/18/2022 10:02:22 PM - 	Version            : 1.12.0.0
4/18/2022 10:02:22 PM - 	Description        : GitHub Copilot is an AI pair programmer that helps you write code faster and with less work.
4/18/2022 10:02:22 PM - 	Locale             : en-US
4/18/2022 10:02:22 PM - 	MoreInfoURL        : 
4/18/2022 10:02:22 PM - 	InstalledByMSI     : False
4/18/2022 10:02:22 PM - 	SupportedFrameworkVersionRange : [0.0,2147483647.2147483647]
4/18/2022 10:02:22 PM - 
4/18/2022 10:02:22 PM - 	SignatureState     : ValidSignature
4/18/2022 10:02:22 PM - 	SignedBy           : GitHub, Inc.
4/18/2022 10:02:22 PM - 	Certificate Info   : 
4/18/2022 10:02:22 PM - 		-------------------------------------------------------
4/18/2022 10:02:22 PM - 		[Subject]       : CN="GitHub, Inc.", O="GitHub, Inc.", L=San Francisco, S=California, C=US
4/18/2022 10:02:22 PM - 		[Issuer]        : CN=DigiCert SHA2 Assured ID Code Signing CA, OU=www.digicert.com, O=DigiCert Inc, C=US
4/18/2022 10:02:22 PM - 		[Serial Number] : 09144B52C8330F257D2512DAAAA1E545
4/18/2022 10:02:22 PM - 		[Not Before]    : 2/4/2020 6:00:00 PM
4/18/2022 10:02:22 PM - 		[Not After]     : 2/9/2023 6:00:00 AM
4/18/2022 10:02:22 PM - 		[Thumbprint]    : 9C5A6D389E1454F2ED9EE9419CDF743689709F9C
4/18/2022 10:02:22 PM - 
4/18/2022 10:02:22 PM - 	Supported Products : 
4/18/2022 10:02:22 PM - 		Microsoft.VisualStudio.Community
4/18/2022 10:02:22 PM - 			Version : [17.1,18.0)
4/18/2022 10:02:22 PM - 			ProductArchitecture : amd64
4/18/2022 10:02:22 PM - 
4/18/2022 10:02:22 PM - 	References         : 
4/18/2022 10:02:22 PM - 		-------------------------------------------------------
4/18/2022 10:02:22 PM - 		Identifier   : Microsoft.VisualStudio.IntelliCode.Internal.InlineSuggestions
4/18/2022 10:02:22 PM - 		Name         : Inline suggestions
4/18/2022 10:02:22 PM - 		Version      : [2.2.1930.216,3.0)
4/18/2022 10:02:22 PM - 		MoreInfoURL  : 
4/18/2022 10:02:22 PM - 		Nested       : Yes
4/18/2022 10:02:22 PM - 
4/18/2022 10:02:22 PM - 	Prerequisites      : 
4/18/2022 10:02:22 PM - 		-------------------------------------------------------
4/18/2022 10:02:22 PM - 		Identifier   : Microsoft.VisualStudio.Component.IntelliCode
4/18/2022 10:02:22 PM - 		Name         : IntelliCode
4/18/2022 10:02:22 PM - 		Version      : [17.0,)
4/18/2022 10:02:22 PM - 
4/18/2022 10:02:22 PM - Signature Details...
4/18/2022 10:02:22 PM - 	Extension is signed with a valid signature.
4/18/2022 10:02:22 PM - 
4/18/2022 10:02:22 PM - Searching for applicable products...
4/18/2022 10:02:22 PM - Found installed product - Global Location
4/18/2022 10:02:22 PM - Found installed product - Visual Studio Enterprise 2019 (2)
4/18/2022 10:02:22 PM - Found installed product - Visual Studio Enterprise 2022
4/18/2022 10:02:22 PM - VSIXInstaller.NoApplicableSKUsException: This extension is not installable on any currently installed products.
   at VSIXInstaller.ExtensionService.GetInstallableDataImpl(IInstallableExtension extension, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionService.GetInstallableData(String vsixPath, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionPackService.IsExtensionPack(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.ExtensionPackService.ExpandExtensionPackToInstall(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize(Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize()
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)

**Comment by damonous** [comment]
I'm receiving the exact same error.  It doesn't show up in the marketplace, and I get the same error when I try to install it after download.

Microsoft Visual Studio Professional 2022 (64-bit) - LTSC 17.4 Version 17.4.8

What's the workaround here?
